python/activator/activator.py

```diff
@@ -586,10 +732,18 @@ def server_error(e: Exception) -> tuple[str, int]:
 
 
 def main():
-    # This function is only called in test environments. Container
-    # deployments call `create_app()()` through Gunicorn.
-    app = create_app()
-    app.run(host="127.0.0.1", port=8080, debug=True)
+    # Knative deployments call `create_app()()` through Gunicorn.
+    # Keda deployments invoke main.
```

Comment:
Hmm... ok. 😬 Well, maybe we can get rid of Flask runner later, I think K-T is the only one who ever used it.